keylime/whitelists/README.md

```diff
@@ -0,0 +1,10 @@
+# Keylime Whitelist Files
+
+This folder contains the whitelists needed for each machine to be provisioned
+on machine running the keylime agent.
+
+To create a whitelist, run this on the target machine:
```

Comment:
It would be run on the openshift nodes which we will monitor (and hacked to fail). It only needs to be on the verifier and registrar for when we provision.